frame/staking/src/lib.rs

```diff
@@ -3211,8 +2377,7 @@ impl<T: Config> Module<T> {
 		}
 
 		// emit event
-		// TWO_PHASE_NOTE: remove the inner value.
-		Self::deposit_event(RawEvent::StakingElection(ElectionCompute::Signed));
+		Self::deposit_event(RawEvent::StakingElection);
 
 		log!(
```

Comment:
in `select_and_update_validators` this log wasn't executed when current_era is 0